contrib/mesos/pkg/scheduler/components/schedulerloop/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package schedulerloop implement the scheduler loop which waits for pod
+// events from the queuer and passes them to the SchedulerAlgorithm.
+package schedulerloop
```

Comment:
"Controller" isn't too bad because it resembles the controllers in the ControllerManager.